drivers/scheduler/k8s/specs/test-sharedv4/pod.yaml

```diff
@@ -45,6 +45,24 @@ spec:
           mountPath: /test-sharedv4-vol
         - name: local-vol
           mountPath: /local-vol
+      - name: sv4test-reader
+        image: portworx/sharedv4-test:torpedo
+        imagePullPolicy: Always
+        command: ["python", "/app/reader.py"]
+        # sleeping for 9 seconds from 3 * number of pods * seconds
+        args: ["--interval=9", "$(LOCAL_FILE)"]
+        env:
+          - name: MY_POD_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+          - name: LOCAL_FILE
+            value: "/local-vol/$(MY_POD_NAME)"
+        volumeMounts:
+        - name: test-sv4-vol-svc
+          mountPath: /test-sv4-vol-svc
+        - name: local-vol
+          mountPath: /local-vol
```

Comment:
We are mounting 2 vols. Does the reader use both or just one?

Reply:
removed the `test-v4-vol-svc` path